frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistScene.tsx

```diff
@@ -131,25 +131,22 @@ export function SessionRecordingsPlaylistScene(): JSX.Element {
                     </>
                 }
             />
-            {playlist.short_id && pinnedRecordings !== null ? (
-                <div className="SessionRecordingPlaylistHeightWrapper">
-                    <SessionRecordingsPlaylist
-                        logicKey={playlist.short_id}
-                        // backwards compatibilty for legacy filters
-                        filters={
-                            playlist.filters && isUniversalFilters(playlist.filters)
-                                ? playlist.filters
-                                : convertLegacyFiltersToUniversalFilters({}, playlist.filters)
-                        }
-                        onFiltersChange={setFilters}
-                        onPinnedChange={onPinnedChange}
-                        pinnedRecordings={pinnedRecordings ?? []}
-                        updateSearchParams={true}
-                    />
-                </div>
-            ) : (
-                <NotFound object="Recording Playlist" />
```

Comment:
this causes us to show not found while we load the recordings... we don't need anything here... the playlist handles its own loading state